setup.cfg

```diff
@@ -118,6 +118,10 @@ tests =
     flake8>=3.0.0
     mypy>=0.910,<1.9
     # https://github.com/pytest-dev/pytest-asyncio/issues/706
+    # BACK COMAPAT: pytest-asyncio 0.21.2
+    # FROM: python 3
+    # TO: python 3.7
+    # URL: https://github.com/cylc/cylc-flow/pull/6726
     pytest-asyncio>=0.21.2,!=0.23.*
```

Comment:
We can't drop 0.21 support just yet, because it's the last version to support Python 3.7 :(